tests/Tests/600__ai_apps/660__openvino/660__openvino.robot

```diff
@@ -23,7 +23,7 @@ Verify OpenVino Is Available In RHODS Dashboard Explore Page
   Verify Service Provides "Get Started" Button In The Explore Page    OpenVINO
 
 Verify Openvino Operator Can Be Installed Using OpenShift Console
-   [Tags]  ODS-675   ODS-702  Sanity
+   [Tags]  ODS-675   ODS-702  Tier2
```

Comment:
We should try to keep tags formatting consistent, could you update it to
```
[Tags]  Tier2
...    ODS-XYZ
```
?

Reply:
done